web/src/app/util/GoogleAnalytics.js

```diff
@@ -16,37 +16,23 @@ export function sendGAEvent(eventProps) {
   if (isInitialized) event(eventProps)
 }
 
-class GoogleAnalytics extends Component {
-  componentDidMount() {
-    this.logPageChange(this.props.location.pathname, this.props.location.search)
-  }
-
-  componentDidUpdate({ location: prevLocation }) {
-    const {
-      location: { pathname, search },
-    } = this.props
-    const isDifferentPathname = pathname !== prevLocation.pathname
-    const isDifferentSearch = search !== prevLocation.search
-
-    if (isDifferentPathname || isDifferentSearch) {
-      this.logPageChange(pathname, search)
-    }
-  }
-
-  logPageChange(pathname, search = '') {
+function GoogleAnalytics(props) {
+  const logPageChange = (pathname, search = '') => {
```

Comment:
Looks like we can just put the logic from this function directly in the `useEffect` body